components/mgmt/azure-publc-dns.tf

```diff
@@ -0,0 +1,5 @@
+module "public-dns" {
+  source             = "../../modules/azure-public-dns/"
+  env                = var.env
+  project            = var.project
```

Comment:
doesn't this need the zones argument?

and records?

Reply:
No, doesn't need the zones argument. That's currently picked up in the dns-zones.auto.tfvars (any zones referenced in recordsets in dns-records.auto.tfvars - this behaviour will be changed). In some case, this is made explicit but I've removed the need to do so here

Reply:
I don't understand this, isn't this the place to add the zone and records config?